### PR TITLE
Various Patches

### LINKED ISSUE #643: Can build torp in hand (and also throw them)
## Round ID:
Not relevant
## Testmerges:
Not relevant
## Reproduction:
Step 1: grab 15 metal
Step 2: make torp fram
Step 3: Congratulations! you now have a torp in your hand, feel free to put it in your backpack and never take it out again because it's too heavy to pick up.
Step 4 (optional): when the torp finishes building it will spawn it on the ground and remove it from your inventory.
Step 5: if you drag a torp onto a munitions trolley from your backpack you will have a permanent torp in the bottom of your screen where it would be in your backpack. now you can see exactly what state the torp is in at all time! and it never goes away!

### LINKED ISSUE #662: all bridge noises are spammable and hurt my ears ouch ouch ouch
## Round ID:
454

## Testmerges:
irrelevant

## Reproduction:
spam click tactical console
press ctrl while on the overmap in tac console
probably works on other consoles too

### LINKED ISSUE #694: can't zoom out on the overmap in pixel perfect 2x mode
1. Use the scroll wheel while viewing the overmap

2. Nothing happens.

### LINKED ISSUE #863: Runtime in ai-skynet2.dm, line 150: pick() from empty list
Problem Code
```
		if(!potential.len)
			potential = fallback //Nowhere else to go.
		target = pick(potential)
```
Solution: Fix control flow.
Issue File: `nsv13\code\modules\overmap\ai-skynet2.dm`
Full Runtime:
```
[17:38:03] Runtime in ai-skynet2.dm, line 150: pick() from empty list
proc name: move (/datum/fleet/proc/move)
src: 1st Syndicate Defense Force (/datum/fleet/abassi)
call stack:
1st Syndicate Defense Force (/datum/fleet/abassi): move(null, 0)
/datum/callback (/datum/callback): InvokeAsync()
Timer (/datum/controller/subsystem/timer): fire(0)
Timer (/datum/controller/subsystem/timer): ignite(0)
Master (/datum/controller/master): RunQueue()
Master (/datum/controller/master): Loop()
Master (/datum/controller/master): StartProcessing(0)
```

### LINKED ISSUE #875: Torp can't be moved by conveyor
Conveyor layout on Pegasus implies that torp should be able to move along conveyor belts, but they can't.

This is because they have the anchored var set to true.

### PR DESCRIPTION
**Pull request by francinum**
fixes #875
fixes #694
fixes #643 
fixes #863 
probably fixes #662 

## Changelog
:cl:
fix: Torpedo-derived ammo now respects conveyor belts.
refactor: Overmap-related zooming is now less fundamentally awful, now properly disrespecting pixel scale options.
fix: Torpedo frames are now built on the ground, as well as being subject to conveyor belts.
sounddel: The bridge crew can no longer deafen the entire ship by spam-cycling their firemodes.
/:cl:

### CODE REVIEW

File: nsv13/code/modules/client/zoom.dm [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+		if(delta_y > 0)
+			overmap_zoomout--
+		else
+			overmap_zoomout++
+
+
+		overmap_zoomout = CLAMP(overmap_zoomout, 0, 15)
+		rescale_view(overmap_zoomout, 0, ((40*2)+1)-15)
+
+/* //Oh my god KMC all of this is A W F U L
```
Comment (francinum) [inline nsv13/code/modules/client/zoom.dm:18]: I'm leaving this here, purely so it can be remembered how badly you over-engineered this shit.

Comment (Kmc2000) [inline nsv13/code/modules/client/zoom.dm:18]:
```suggestion
/* //Oh my god KMC all of this is A W F U L - I know, right? ~K
```

File: nsv13/code/modules/munitions/ammunition/torpedos/torpedo_types.dm [hunk 2]
```diff
@@ -3,14 +3,15 @@
 	icon = 'nsv13/icons/obj/munition_types.dmi'
 	icon_state = "standard"
 	desc = "A fairly standard torpedo which is designed to cause massive structural damage to a target. It is fitted with a basic homing mechanism to ensure it always hits the mark."
-	anchored = TRUE
 	density = TRUE
+	move_resist = MOVE_FORCE_EXTREMELY_STRONG
```
Comment (Kmc2000) [inline nsv13/code/modules/munitions/ammunition/torpedos/torpedo_types.dm:7]: That's a nice way of doing things, actually

**Review by Kmc2000 — APPROVED** [review]
This is pretty nice, good work!

**Pull request merged**